modules/combined/test/tests/optimization/invOpt_elasticity/forward.i

```diff
@@ -0,0 +1,193 @@
+[GlobalParams]
+  displacements = 'ux uy'
+[]
+
+[Mesh]
+  [gmg]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 11
+    ny = 11
+    xmin = -4
+    xmax = 4
+    ymin = -4
+    ymax = 4
+  []
+[]
+
+[AuxVariables]
+  [dummy]
+  []
```

Comment:
What's the use of this dummy variable?

Reply:
Good catch. It is actually not used in here.